src/webSocket/getAutoSuggestUsers.ts

```diff
@@ -0,0 +1,16 @@
+import WebSocket from 'ws';
+
+import { server } from '../index';
+import { filterUserByLogin } from '../controllers/user';
+
+export const getAutoSuggestUsers = () => {
+  const wss = new WebSocket.Server({ server });
+
+  wss.on('connection', (ws: WebSocket) => {
+    ws.on('message', (message: string) => {
```

Comment:
Callbacks can be extracted as separate functions

Reply:
fixed